conf/pacific/upgrades/tier-2_10node_cephadm_upgrade_scale.yaml

```diff
@@ -0,0 +1,72 @@
+globals:
```

Comment:
Could we opt to describe the layout? For example: 
```
10-node-cluster-with-5-pools.yaml
10-node-cluster.yaml
```

Reply:
Done.